web/html/links-ii/image-mailto.md

```diff
@@ -30,17 +29,57 @@ Creating a link to open the user's default email program:
 
 [View CodePen](https://codepen.io/enkidevs/pen/ejvqXx)
 
-The subject area of the email can also be auto loaded.
+`mailto:` has 4 attributes; cc, bcc, subject, and body.
+
+Attributes are added by writing their name followed by an equal sign and the attribute's value.
+
+> 💡 When adding the subject and/or the body, rather than using spaces between words it is recommended to write `%20`[1] in place of each space. 
```

Comment:
I think we can add a small phrase here which explains why it's not good to include spaces, and if users are more interested they can open the footnote

I've also suggested updating the footnote's headline to URL Syntactic Rules

```suggestion
> 💡 When adding the subject and/or the body, rather than using spaces between words it is recommended to write `%20` in place of each space. This is because URLs have strict syntactic rules[1].
```